e2e/backend.e2e.spec.ts

```diff
@@ -0,0 +1,228 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+
+import _ from 'lodash';
+import { ElectronApplication, BrowserContext, _electron, Page } from 'playwright';
+import { test, expect } from '@playwright/test';
+
+import { createDefaultSettings, packageLogs, reportAsset } from './utils/TestUtils';
+import { NavPage } from './pages/nav-page';
+import { Settings, ContainerEngine } from '@/config/settings';
+import fetch from '@/utils/fetch';
+import paths from '@/utils/paths';
+import { RecursivePartial } from '@/utils/typeUtils';
+
+type KubeSettings = Settings['kubernetes'];
+
+test.describe.serial('KuberentesBackend', () => {
```

Comment:
Typo in "Kubernetes..."

Reply:
Fixed in 8d4efdc31269e66cef6d53a04cef885e85b14ef1.